gremlin-javascript/src/main/javascript/gremlin-javascript/lib/process/graph-traversal.js

```diff
@@ -543,6 +543,16 @@ class GraphTraversal extends Traversal {
     return this;
   }
 
+  /**
```

Comment:
i don't believe this is the only change required for javascript. you need the `statics` method definition as well